### DISCUSSION TITLE
Preview Prompt Conditioning

### DISCUSSION THREAD

**Discussion by DaveTheDayTrader** [comfyanonymous/ComfyUI#3945] [Q&A]
When I write a prompt and also use a prompt style, I have no idea what the final result looks like. How much does my prompt merge into the styler prompt, does it get modified or even used ( overridden?)

I thought once I saw  someone create like a text box that shows the final prompt that is being used - or was that a dream?
lol

Anyone know of a way to show this info?

Thx

**Comment by ltdrdata** [comment]
There is no way to preview through an arbitrary prompt that you input. The closest thing to that is live generation.
Another method is to use the predefined prompt style node that comes with a preview.

![sample_2](https://github.com/comfyanonymous/ComfyUI/assets/128333288/45eb4ba9-f78e-4394-8c54-d17864670788)
https://github.com/syllebra/bilbox-comfyui

**Comment by JorgeR81** [comment]
Could, theoretically, be possible to have a 1-4 step lora, that looks similar to a 20 - 30 steps checkpoint ?
With less detail, but the same overall image. 
If so, it could be used to generate a small size "preview". 
The turbo / lightning / hyper loras we have right now give good looking results, but the final image it's quite different.

**Reply by ltdrdata** [reply]
That's not possible. Even resolution, seed, sampler, cfg - all these factors completely change the resulting image. It can only help to roughly understand what kind of image that particular condition might produce using the lowest-cost image generation workflow.

**Reply by mcmonkey4eva** [reply]
you can use the turbo result as the init image for the full gen, and that will make it match better

**Reply by JorgeR81** [reply]
OK, I see. 

Ironically, the now mostly forgotten Stable Cascade model, could be good for this.

With Sable Cascade, if we use a VAE after stage C, it would give a good miniature image of what's stage B is going to look like.

If stage C could be improved to give a good result after 1-2 steps, it could work as a preview.
